### PR TITLE
Corrected step reference

### PR DESCRIPTION
**Pull request by MarcinOrlowski**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # This workflow will build a Java project with Gradle
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+# Using lowest supported JSK (11) and highest available (16)
```
Comment (maehne) [inline .github/workflows/build.yml:2]: Typo: JSK -> JDK

**Review by maehne — COMMENTED** [review]
I think a small typo is left in the first modified comment.

File: .github/workflows/build.yml (hunk 1 above)
Comment (MarcinOrlowski) [reply .github/workflows/build.yml:2]: Good spot

Comment (MarcinOrlowski) [reply .github/workflows/build.yml:2]: Fixed

**Pull request merged**